src/runtime/c_runtime_api.cc

```diff
@@ -474,7 +474,8 @@ int TVMFuncCreateFromCFunc(TVMPackedCFunc func, void* resource_handle, TVMPacked
       int ret = func(const_cast<TVMValue*>(args.values), const_cast<int*>(args.type_codes),
                      args.num_args, rv, resource_handle);
       if (ret != 0) {
-        throw dmlc::Error(TVMGetLastError() + ::dmlc::StackTrace());
+        throw tvm::Error(TVMGetLastError() +
+                         tvm::runtime::Backtrace());  // TODO(tkonolige): this may double backtrace
```

Comment:
can you explain more when and what is left todo?

Reply:
TVMGetLastError may already contain a backtrace, so this would double it. (This was a preexisting issue).

Reply:
can you comment this in? why would it already contain a backtrace--because this function was previously called down-stack?

Reply:
I think my todo is wrong. This is just appending the stack trace to the existing one.